docker/Dockerfile

```diff
@@ -14,20 +14,21 @@ RUN apt-get update && \
         python3 \
         python3-dev \
         python3-pip \
-        postgresql-client
+        postgresql-client \
+	wget
 
 ADD . /app
 WORKDIR /app
 
-RUN pip3 install -U pip
-RUN pip3 install -i https://pypi.pacificclimate.org/simple/ -r requirements.txt
-RUN pip3 install .
+RUN wget -O - https://install.python-poetry.org | python3 -
+ENV PATH=/root/.local/bin:$PATH
+RUN poetry install
 
 EXPOSE 8000
 
-ENV FLASK_APP wxfs.wsgi
+ENV FLASK_APP wxfs.asgi
 ENV FLASK_ENV development
 ENV LC_ALL C.UTF-8
 ENV LANG C.UTF-8
 
-CMD flask run -p 8000 -h 0.0.0.0 --no-reload
+CMD poetry run uvicorn --port 8000 --host 0.0.0.0 --reload wxfs.asgi:connexion_app
```

Comment:
Yay, sooo much better to serve it with something proper. I can't believe I didn't use Gunicorn before.